### ISSUE TITLE
Joystick is not detected in v3.5.0 mac release

### ISSUE THREAD

**Issue by patrickelectric** [mavlink/qgroundcontrol#7252]
## Expected Behavior
Detect the joystick.

## Current Behavior
Do not detected joystick.

## Steps to Reproduce:
1. Connect joystick
2. Open QGC
3. Device is not detected

## System Information
- Operating System: 10.14.2
- QGC Version: 3.5.0

![image](https://user-images.githubusercontent.com/1215497/53440443-6358ae80-39b9-11e9-88ff-74cd71c5038b.png)

[log.txt](https://github.com/mavlink/qgroundcontrol/files/2907167/log.txt)

**Comment by DonLakeFlyer** [comment]
QGC uses SDL lib for joysticks. So if SDL doesn't see it QGC won't see it. Only possible fix is is newer version of SDL can see it.

**Comment by DonLakeFlyer** [comment]
Or is this a new problem where it was working before?

**Comment by patrickelectric** [comment]
Yes, it was working before, but since I did some updates in the OS, that may be the cause.
